dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -698,7 +698,7 @@ HRESULT CFSDropTarget::_DoDrop(IDataObject *pDataObject,
         _ILFreeaPidl(apidl, lpcida->cidl);
         ReleaseStgMedium(&medium);
     }
-    else if (SUCCEEDED(pDataObject->QueryGetData(&fmt2)))
+    else if (SUCCEEDED(hr = pDataObject->QueryGetData(&fmt2)))
     {
         FORMATETC fmt2;
         InitFormatEtc (fmt2, CF_HDROP, TYMED_HGLOBAL);
```

Comment:
Suggestion to do: `hr = pDataObject->GetData(&fmt2, &medium)` for the call below...